pom.xml

```diff
@@ -15,8 +15,8 @@
         <maven>3.0.5</maven>
     </prerequisites>
 
-    <groupId>com.googlecode.maven-java-formatter-plugin</groupId>
-    <artifactId>maven-java-formatter-plugin</artifactId>
+    <groupId>com.github.java-formatter-maven-plugin</groupId>
```

Comment:
If you have the time, can you make this change and cancel this PR?  I've got a few other things to deal with so will be a few days for me to get back to this.